babel.config.js

```diff
@@ -28,7 +28,8 @@ module.exports = api => {
     plugins: [
       '@babel/plugin-proposal-class-properties',
       '@babel/plugin-proposal-export-default-from',
-      '@babel/plugin-syntax-dynamic-import'
+      '@babel/plugin-syntax-dynamic-import',
+      '@babel/plugin-transform-runtime'
```

Comment:
what's this new babel plugin required for?

Reply:
This is me trying to get past the test failures I'm seeing atm